src/cljs/chronograph_web/subscriptions.cljs

```diff
@@ -10,3 +10,23 @@
   ::user-info
   (fn [db _]
     (:user db)))
+
+(rf/reg-sub
+  ::current-page
+  (fn [db _]
+    (:page db)))
+
+(rf/reg-sub
+  ::create-organization-form
+  (fn [db _]
+    (get-in db [:create-organization])))
+
+(rf/reg-sub
+  ::organization-from-slug
+  (fn [db [_ slug]]
+    (get-in db [:organizations-by-slug slug])))
```

Comment:
As mentioned earlier, this can just do a linear search. Let's avoid duplicating state unless absolutely necessary.

Reply:
We may need to do this later. For now, we've decided to index organizations by slug.